Dockerfile

```diff
@@ -4,6 +4,8 @@ MAINTAINER f99aq8ove <f99aq8ove [at] gmail.com>
 
 ADD https://github.com/gitbucket/gitbucket/releases/download/4.18.0/gitbucket.war /opt/gitbucket.war
 
+apk add --no-cache ttf-dejavu git
```

Comment:
[The gitbucket plugin (commit graphs)](https://github.com/yoshiyoshifujii/gitbucket-commitgraphs-plugin) needs `git` package. (That plugin should be install manually)

Reply:
I think that should be "RUN apk add ...".

Reply:
Oops! Sorry for mistake. 
You are right.
Thank you reviewing.